### PR TITLE
fix: id:46394 Icon component accessibility

### PR DESCRIPTION
**Pull request by Farhan-emgage**
Icon accesibility with accesibilityLabel. included role and tabIndex for achieve focus

### CODE REVIEW

**Comment by emgageadmin** [comment]
https://atera.tpondemand.com/entity/46394

File: src/components/Drawer/Drawer.scss [hunk 1]
```diff
@@ -280,7 +280,7 @@ body.overlay {
 }
 
 body.container {
-  overflow: auto !important;
+  overflow: visible !important;
```
Comment (weegrom) [inline src/components/Drawer/Drawer.scss:283]: @Farhan-emgage has this been tested to confirm it does not create an issue with wide content displaying outside the drawer?

**Review by weegrom — CHANGES_REQUESTED** [review]
@Farhan-emgage please respond to my 1 comment, and also resolve merge conflict. Thanks

File: src/components/Drawer/Drawer.scss (hunk 1 above)
Comment (Farhan-emgage) [reply src/components/Drawer/Drawer.scss:283]: @weegrom yes i tested it. when the drawer is open at that time bodu.container class is apply on body and when drawer is close then this class is removed

**Comment by weegrom** [comment]
@Farhan-emgage please resolve the merge conflict here so we can get this PR merged.

**Pull request closed without merging**